extensions/amp-list/amp-list.md

```diff
@@ -578,3 +578,5 @@ The AMP for Email spec disallows the use of the following attributes on the AMP
 ## Validation
 
 See [amp-list rules](https://github.com/ampproject/amphtml/blob/master/extensions/amp-list/validator-amp-list.protoascii) in the AMP validator specification.
+
+[/filter]
```

Comment:
Does this need to be changed both here and in #29550?

Reply:
Nope, I just didn't see Patrick's PR 😁  both will do the trick and unblock the amp.dev's build

Reply:
Oh I see. :) There I recommended removing the opening tag instead of adding the closing tag.

Reply:
Done!